### DISCUSSION TITLE
import espnow can not be imported

### DISCUSSION THREAD

**Discussion by wajidali4** [micropython/micropython#12327] [ESP32]
Hello, i am new to micropython and I am using esp wroom 32 and Thonny IDE but espnow module can not be imported ,can you please guide me how to fix this issue.

**Comment by davefes** [comment]
Which Micropython image are you putting on the ESP32?

**Reply by wajidali4** [reply]
i downloaded this from micropython website esp32-20230426-v1.20.0.bin

**Reply by davefes** [reply]
I tried to find a recent thread which explained the changes that have happened with ESPNow recently.

I believe you need a nightly build to get an up to date ESPNow.

For ESP32 Generic try nightly build:
v1.20.0-396-g1dedb65e6.bin

**Reply by wajidali4** [reply]
can you share me the link please

**Reply by robert-hh** [reply]
https://micropython.org/download/ESP32_GENERIC/
It's the official MicroPython firmware download site.

**Reply by wajidali4** [reply]
thank you all for support now i am able to import espnow by flashing nightly build:
v1.20.0-396-g1dedb65e6.bin
